### DISCUSSION TITLE
CODE REVIEW IM-11

### DISCUSSION THREAD

**Discussion by YevheniiSemyvolos** [HowProgrammingWorks/LiveQA#48] [Code review]
Пишу браузерну гру: "Let's Throw Dice!".

Суть гри:
Грають два гравця. Мета кожного: зібрати всі комбінації кубиків (від Chance до Poker).
Усього 8 раундів. У кожному раунді гравець може кинути до 3-х разів 5 гральних кубиків заради отримання кращої комбінації.
Якщо гравець вирішив зупинитися на даній комбінації, він записує кількість очок, отриманих за комбінацію. Одній комбінації відповідає один результат. Наприклад, якщо комбінація _Straight_ вже має результат у гравця, то гравець не має права перезеписати результат. Треба обрати іншу комбінацію, де буде результат 0, або перекинути кубики.
Очки обчислюються як сума всіх кубиків, що входять у комбінацію, + бонус за комбінацію.
Виграє той гравець, хто має більш поінтів.
Буду вдячній за ревью [кода](https://github.com/EugeneSemivolos/LetsThrowDice/tree/dev).

**Comment by tshemsedinov** [comment]
Lonc to review1: https://github.com/EugeneSemivolos/LetsThrowDice/issues/2
